modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shibari_worn.dm

```diff
@@ -0,0 +1,268 @@
+/obj/item/clothing/under/shibari
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_clothing/lewd_uniform.dmi'
+	worn_icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_uniform/lewd_uniform.dmi'
+	worn_icon_digi = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_uniform/lewd_uniform-digi.dmi'
+	worn_icon_taur_snake = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_uniform/lewd_uniform-snake.dmi'
+	worn_icon_taur_paw = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_uniform/lewd_uniform-paw.dmi'
+	worn_icon_taur_hoof = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_uniform/lewd_uniform-hoof.dmi'
+	can_adjust = FALSE
+	body_parts_covered = NONE
+	strip_delay = 100
+	mutant_variants = STYLE_DIGITIGRADE|STYLE_TAUR_ALL
+	item_flags = DROPDEL
+
+	///colors
+	var/current_color = "pink"
+	///tightness of the ropes can be low, medium and hard.
+	var/tight = "low"
```

Comment:
Should be called `tightness` and that value should be a define, like, `ROPE_TIGHTNESS_LOW`, etc.